deepchecks/nlp/utils/text_properties.py

```diff
@@ -192,18 +201,40 @@ def max_word_length(raw_text: Sequence[str]) -> List[int]:
     return [max([len(word) for word in text.split()]) for text in raw_text]
 
 
-def language(raw_text: Sequence[str]) -> List[str]:
+def language(raw_text: Sequence[str],
+             models_storage: Union[pathlib.Path, str, None] = None,
+             lang_certainty_threshold: float = 0.8
+             ) -> List[str]:
     """Return list of strings of language."""
-    langdetect = _import_optional_property_dependency(module='langdetect', property_name='language')
-    langdetect.DetectorFactory.seed = 42
+    fasttext = _import_optional_property_dependency(module='fasttext', property_name='language')
 
-    result = []
-    for text in raw_text:
-        try:
-            result.append(langdetect.detect(text))
-        except langdetect.lang_detect_exception.LangDetectException:
-            result.append(np.nan)
-    return result
+    model_name = FASTTEXT_LANG_MODEL.rsplit('/', maxsplit=1)[-1]
+
+    model_path = get_creat_model_storage(models_storage)
+    model_path = model_path / 'fasttext'
+    model_path = get_creat_model_storage(model_path)
+    model_path = model_path / model_name
```

Comment:
this confused me, cant you do:
get_creat_model_storage(models_storage / 'fasttext')?